xbmc/utils/FileUtils.cpp

```diff
@@ -53,7 +54,7 @@ bool CFileUtils::DeleteItem(const CFileItemPtr &item, bool force)
   if (!item || item->IsParentFolder())
     return false;
 
-  CGUIDialogYesNo* pDialog = g_windowManager.GetWindow<CGUIDialogYesNo>(WINDOW_DIALOG_YES_NO);
+  CGUIDialogYesNo* pDialog = CServiceBroker::GetGUI()->GetWindowManager().GetWindow<CGUIDialogYesNo>(WINDOW_DIALOG_YES_NO);
```

Comment:
this breaks the test suite

Reply:
once again the useless test suite. can you point me to the last known case where the test suite was useful by bringing an issue to surface?

Reply:
but this is a good opportunity to drop a dependency from a utility to gui. I'll fix it